lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -194,21 +199,25 @@ def _get_job_log(self, job_id, skip_lines):
         :param skip_lines:
         :return:
         """
-        log = self.get_mongo_util().get_job_log(job_id)
+
+        mongo_util = self.get_mongo_util()
+        with mongo_util.me_collection(self.config["mongo-logs-collection"]):
```

Comment:
do we need this "with" statement here? Isn't there one inside the `get_job_log_pymongo` function? If we need it, can we move this with statement into that function?

Reply:
👍